### PR TITLE
FAQ: Audit SSH usage

### PR DESCRIPTION
**Pull request by drnickiw**
## what
* How to audit SSH usage FAQ

## why
* Imported from Google Doc

Links to #353

### CODE REVIEW

File: content/faq/how-to-audit-ssh-usage.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+## Question
+
+How do we audit SSH usage and track what is done on a host?
+
+## Answer
+
+The best way is with Teleport. We’ve implemented this with other clients. However, those Helm Charts are not yet open-sourced. We are working with them to make that happen; see [here](https://github.com/gravitational/teleport
+).
```
Comment (osterman) [inline content/faq/how-to-audit-ssh-usage.md]:
```suggestion
```

File: content/faq/how-to-audit-ssh-usage.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- Helm Chart
+---
+
+## Question
+
+How do we audit SSH usage and track what is done on a host?
+
+## Answer
+
+The best way is with Teleport. We’ve implemented this with other clients. However, those Helm Charts are not yet open-sourced. We are working with them to make that happen; see [here](https://github.com/gravitational/teleport
```
Comment (osterman) [inline content/faq/how-to-audit-ssh-usage.md]:
```suggestion
The best way is with Teleport. We’ve implemented this with other clients. However, those Helm Charts are not yet open-sourced. We are working with them to make that happen; see [here](https://github.com/gravitational/teleport).
```

File: content/faq/how-to-audit-ssh-usage.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+).
+
+Gravitational makes a Helm Chart available for Teleport. However, last we checked, it didn't work the way we'd want it to, whereby Teleport is deployed on all nodes as a `DaemonSet`.
```
Comment (osterman) [inline content/faq/how-to-audit-ssh-usage.md]:
```suggestion
Gravitational makes a [Helm Chart available for Teleport](https://github.com/gravitational/teleport/tree/master/examples/chart/teleport). However, last we checked, it didn't work the way we'd want it to, whereby Teleport is deployed on all nodes as a `DaemonSet`.
```

File: content/faq/how-to-audit-ssh-usage.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Technically, we have our own solution called `sudosh`, but that's subpar by comparison. It's an extremely simple wrapper for `sudo` that enables it to be used as a system login shell. `sudo` natively supports session logs and replay. The downside with this solution is we still must store the `sudo` binary logs somewhere, so it's not as tamper-resistant as Teleport. In addition, the logs are binary, so shipping them to a log store like Sumologic or Splunk is not recommended.
```
Comment (osterman) [inline content/faq/how-to-audit-ssh-usage.md]:
```suggestion
Technically, we have our own solution called [`sudosh`](https://github.com/cloudposse/sudosh), but that's subpar by comparison. It's an extremely simple wrapper for `sudo` that enables it to be used as a system login shell. `sudo` natively supports session logs and replay. The downside with this solution is we still must store the `sudo` binary logs somewhere, so it's not as tamper-resistant as Teleport. In addition, the logs are binary, so shipping them to a log store like Sumologic or Splunk is not recommended.
```

**Comment by osterman** [comment]
This PR includes changes from https://github.com/cloudposse/docs/pull/385

This usually means one of two things:
1) Your `master` branch is not pristine (as in contains changes), so if you fork from it, you're including changes. See https://stackoverflow.com/a/1628334/1237191
2) You are not changing back to `master` before starting a new fork

**Comment by drnickiw** [comment]
I created new forks from master each time. This first one stated there was a difference in the commits that was awaiting review from a reviewer at the origin though, so I assumed it was something that was pending on your end and continued on. I'll check this issue though.

________________________________

**Comment by drnickiw** [comment]
> This PR includes changes from #385
> 
> This usually means one of two things:
> 
> 1. Your `master` branch is not pristine (as in contains changes), so if you fork from it, you're including changes. See https://stackoverflow.com/a/1628334/1237191
> 2. You are not changing back to `master` before starting a new fork

**Comment by drnickiw** [comment]
The second unnecessary file should be properly removed.

**Pull request merged**